docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+        </ul>
+    </td>
+    <td>Filters the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a> by the contributions made by the team</td>
+</tr>
+</table>
+
+<sup><span id="commit-author-date">1</span></sup>: Pulse uses the commit author's date since it is more accurate. The committer date can be changed (e.g.: fast-forward) and stop reflecting the real creation date of the change.
+
+<sup><span id="deployment-teams">2</span></sup>: Adding a new user group or changing the composition of a user group on Bitbucket only affects new data from that moment onwards and doesn't have an immediate impact on the dashboards. Also, deleted user groups on Bitbucket are still visible in Pulse.
+
+## Which permissions does Pulse need from Bitbucket? {: id="bb-permissions"}
+
+Pulse requests only the necessary [permissions/scopes from Bitbucket](https://developer.atlassian.com/cloud/bitbucket/bitbucket-cloud-rest-api-scopes/) to collect changes and deployment data from the repositories in your workspace and [keeps your information secure](https://security.codacy.com/). See below the detailed list of permissions.
+
+<table>
+  <colgroup>
+    <col width="25%"/>
+    <col width="75%"/>
+  </colgroup>
+  <thead>
+    <tr>
+      <th>Scope and permissions</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>account</code></td>
+      <td>Pulse retrieves the email of the user installing the integration, the user's workspaces and its membership, and the user groups.</td>
+    </tr>
+    <tr>
+      <td><code>repository</code></td>
+      <td>Pulse retrieves repository information to get the default branch and the list of commits.</td>
+    </tr>
+    <tr>
+      <td><code>pullrequest</code></td>
+      <td>Pulse retrieves pull request information to calculate several metrics presented on the dashboards. <a href="#collected-data">See the details here.</a></td>
+    </tr>
+    <tr>
+      <td><code>webhook</code></td>
+      <td>Pulse creates webhooks to track new or deleted workspaces.
+      <br/>Pulse creates webhooks subscribed to the following events as a trigger to gather the corresponding data in real time:
+      <ul>
+        <li><code>pullrequest:created</code>: pull requests creation</li>
+        <li><code>pullrequest:updated</code>: pull requests edition</li>
+        <li><code>pullrequest:fulfilled</code>: pull requests merge</li>
+        <li><code>pullrequest:rejected</code>: pull requests decline</li>
+        <li><code>pullrequest:approved</code>: pull requests approval</li>
+        <li><code>pullrequest:unapproved</code>: deletion of pull requests approval</li>
+        <li><code>pullrequest:changes_request_created</code>: pull requests change request</li>
+        <li><code>pullrequest:changes_request_removed</code>: deletion of pull requests change request</li>
+        <li><code>repo:push</code>: pushes of commits to a repository</li>
```

Comment:
This event is trigger with pushing branches, commits, tags to a repository, but we only care about tags. Not sure if that is worth mentioning.

Reply:
Note to self: consider editing as below.

```suggestion
        <li><code>repo:push</code>: pushes of commits, branches, or tags to a repository</li>
```